src/gt4py/next/iterator/embedded.py

```diff
@@ -197,7 +199,8 @@ class Column(np.lib.mixins.NDArrayOperatorsMixin):
     def __init__(self, kstart: int, data: np.ndarray | Scalar) -> None:
         self.kstart = kstart
         assert isinstance(data, (np.ndarray, Scalar))  # type: ignore # mypy bug
-        self.data = data if isinstance(data, np.ndarray) else np.full(len(_column_range), data)  # type: ignore[arg-type]
+        _column_range = column_range_cvar.get()
```

Comment:
```suggestion
        column_range = column_range_cvar.get()
```
Since it is a local variable now the underscore is not needed anymore right?